packages/models-library/tests/test_utils_pydantic_tools_extension.py

```diff
@@ -14,15 +14,20 @@ class MyModel(BaseModel):
 
 
 def test_schema():
-    assert MyModel.schema() == {
+    assert MyModel.model_json_schema() == {
         "title": "MyModel",
         "type": "object",
         "properties": {
             "a": {"title": "A", "type": "integer"},
-            "b": {"title": "B", "type": "integer"},
+            "b": {"anyOf": [{"type": "integer"}, {"type": "null"}], "title": "B"},
             "c": {"title": "C", "default": 42, "type": "integer"},
-            "d": {"title": "D", "type": "integer"},
+            "d": {
+                "anyOf": [{"type": "integer"}, {"type": "null"}],
+                "default": None,
+                "title": "D",
+            },
             "e": {
+                "default": None,
```

Comment:
I just wonder whether we could change `FieldNotRequired` to returns some `NotSet` -like value